### DISCUSSION TITLE
Should we enforce decimals for whole numbers in our calculators?

### DISCUSSION THREAD

**Discussion by thomasMatson** [thithuongmai/T24S2T1EXTG05#2] [General]
i.e convert any integers entered into floats, for example:
user enters 5
calculator converts to 5.00

**Comment by thithuongmai** [comment]
- For Currency Conversion: Yes, enforce two decimal places, as currencies often require precision to two decimal points.
- For Mortgage Calculator: I may allow whole numbers, but ensure the output is formatted correctly with two decimal places for consistency in financial calculations.
